todocore/src/main/java/com/knighttodo/todocore/service/DayService.java

```diff
@@ -53,4 +62,13 @@ public void deleteById(UUID dayId) {
         dayRepository.softDeleteAllDayTodosByDayId(dayId);
         dayRepository.softDeleteById(dayId);
     }
+
+    @Transactional
+    public DayVO findDayByDate(LocalDate date) {
+        return dayRepository.findDaysByDate(date).map(dayMapper::toDayVO).orElseThrow(() -> {
+            log.error(String.format("Day with such date:%s can't be " + "found", date));
```

Comment:
add spaces before and after ":" please

Reply:
And remove a redundant + between be and found, it's one string

Reply:
Done!